automated install/basic-install.sh

```diff
@@ -694,10 +694,11 @@ chooseInterface() {
             status="OFF"
         done
         # Disable check for double quote here as we are passing a string with spaces
+        # shellcheck disable=SC2086
```

Comment:
> Directives that replace or are immediately after the shebang apply to the entire script.
> **Otherwise, they are scoped to the command that follows it.** (including compound commands like function definitions, loops and case statements)

That seems okay. Taken from https://github.com/koalaman/shellcheck/wiki/Directive